spec/pom.xml

```diff
@@ -60,7 +62,11 @@
                         </goals>
                         <configuration>
                             <backend>pdf</backend>
-                            <outputFile>${project.build.finalName}.pdf</outputFile>
+                            <attributes>
```

Comment:
You'll need to duplicate this section for the HTML output as well.

Reply:
done. thanks. I copied what @pilhuhn has done for metrics. @pilhuhn you missed this part in the metrics spec/pom.xml as well.